code/src/OSFramework/Grid/AbstractGrid.ts

```diff
@@ -139,16 +140,32 @@ namespace OSFramework.Grid {
                     // Split the binding of the column by every dot. (e.g Sample_product.Name -> ['Sample_Product', 'Name'])
                     const bindingMatches = column.config.binding.split('.');
                     let metadata = this.dataSource.getMetadata();
-                    bindingMatches.forEach((keyword) => {
+
+                    const validate = (keyword, binding) => {
                         // Check if the matching keyword is a property from metadata
                         if (metadata && !metadata.hasOwnProperty(keyword)) {
-                            throw `The binding "${
-                                column.config.binding
-                            }" doesn't match any valid column from the data you specified. ${'\n'} Expected format: "EntityName.FieldName". ${'\n'} For example: "Product_Sample.Name"`;
+                            throw `The binding "${binding}" doesn't match any valid column from the data you specified. ${'\n'} Expected format: "EntityName.FieldName". ${'\n'} For example: "Product_Sample.Name"`;
                         }
                         // If keyword is a property from metadata then use metadata[keyword] as the new metadata and iterate to the next keyword.
                         metadata = metadata[keyword];
-                    });
+                    };
+
+                    bindingMatches.forEach((binding) =>
+                        validate(binding, column.config.binding)
+                    );
+                    // validate dropdown dependency columns
+                    // @ts-ignore
+                    if (column.config.parentBinding) {
```

Comment:
In order to remove ts-ignore, please use column.config.hasOwnProperty(parentBinding)

Reply:
done